dimelo/utils.py

```diff
@@ -7,16 +7,59 @@
 from matplotlib.axes import Axes
 
 # This provides the mapping of canonical bases to sets of valid mode names
-BASEMOD_NAMES_DICT = {
-    "A": {"A", "a", "Y"},
-    "C": {"m", "Z"},
-}
-
-DEFAULT_COLORS = {
-    "A,0": "blue",
-    "CG,0": "orange",
-    "GCH,1": "purple",
-}
+BASEMOD_NAMES_DICT = defaultdict(lambda: set())
```

Comment:
Why is this a default dict? If there is something clever being done here, can you document that choice?